### ISSUE TITLE
When using only PhoneAuthProvider there should be no cancel button

### ISSUE THREAD

**Issue by ThewBear** [firebase/firebaseui-web#413]
There should be the configuration to hide or callback for this case.

**Comment by nicolasgarnier** [comment]
Fair point. @bojeil-google, when PhoneAuthProvider is the only provider displayed it looks like we should hide the "Cancel" button since it's useless.

**Comment by nicolasgarnier** [comment]
Although we might currently need it to refresh the UI when the recaptcha expires...

**Comment by dackers86** [comment]
Thank you for taking the time to submit this issue.

A new release is planned shortly, which should resolve this request. 

In the meantime, we will close this issue, if this is not resolved in the new version - please feel free to create a new issue.
